modules/340-extended-monitoring/openapi/doc-ru-config-values.yaml

```diff
@@ -26,6 +26,11 @@ properties:
               ca:
                 description: |
                   Корневой сертификат, которым можно проверить сертификат container registry при работе по HTTPS (если registry использует самоподписанные SSL-сертификаты).
```

Comment:
```suggestion
                  Корневой сертификат для проверки сертификата container registry при использовании HTTPS (если registry использует самоподписанные SSL-сертификаты).
```

Reply:
The lines of code mentioned in this suggestion were not changed in this Pull Request and are not related to the functionality being added in this Pull Request.